### PR TITLE
Update ingest API version and remove CVE suppression

### PR DESCRIPTION
**Pull request by ahoffer**
- Updates the applications to use version 0.2.0 of the Ion Ingest API. 
- ~~Instructs gradle to also look in the CX PHX nexus for artifacts~~ Already config to use CX PHX nexus
- Removes a CVE suppression that is no longer necessary because the new Ion Ingest API artifacts use a different version of a library.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

File: build.gradle [hunk 1]
```diff
@@ -18,18 +18,25 @@ buildscript {
 }
 
 plugins {
+    id "com.diffplug.gradle.spotless" version "3.23.1"
```
Comment (ahoffer) [inline build.gradle]: Alphabetized

File: build.gradle [hunk 2]
```diff
@@ -18,18 +18,25 @@ buildscript {
...
-    id "com.diffplug.gradle.spotless" version "3.23.1"
     id "org.sonarqube" version "2.7.1"
 }
 
+repositories {
+    maven {
+        url "http://nexus.phx.connexta.com:8081/nexus/content/groups/public"
+    }
+}
+
```
Comment (ahoffer) [inline build.gradle]: Requires VPN or LAN connectivity.

File: build.gradle [hunk 3]
```diff
@@ -18,18 +18,25 @@ buildscript {
...
 allprojects {
 
     sourceCompatibility = Versions.java
     targetCompatibility = Versions.java
 
-    if(sourceCompatibility != Jvm.current().javaVersion) {
+    if (sourceCompatibility != Jvm.current().javaVersion) {
```
Comment (ahoffer) [inline build.gradle]: Either Spotless or IntelliJ decided to make some formatting changes.

File: build.gradle [hunk 4]
```diff
@@ -167,7 +173,9 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/") }
+        maven {
```
Comment (ahoffer) [inline build.gradle]: Something feels like it needs fixing up. I'll look into it.

File: build.gradle (hunk 2 above)
Comment (ahoffer) [reply build.gradle]: Feels like something is duplicated. I'll look into it.

File: build.gradle [hunk 5]
```diff
@@ -18,18 +18,25 @@ buildscript {
...
     id "org.owasp.dependencycheck" version "5.1.1"
-    id "com.diffplug.gradle.spotless" version "3.23.1"
     id "org.sonarqube" version "2.7.1"
 }
 
+repositories {
+    maven {
+        url "http://nexus.phx.connexta.com:8081/nexus/content/groups/public"
+    }
+}
```
Comment (emmberk) [inline build.gradle]: ❓What's the difference between this `repositories` section and the one in `subprojects`? Can we move all of the repositories here?

File: build.gradle [hunk 6]
```diff
@@ -167,8 +162,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/") }
-        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/groups/public/") }
+        maven { url "http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/" }
```
Comment (ahoffer) [inline build.gradle]: I clean out my gradle cache and performed a clean & build with gradle with this URL, which is also used in the Ion Ingest API project. Gradle did not complain about missing repository information. Now both repos use the same nexus URL.

File: ingest/src/main/java/com/connexta/ingest/endpoint/IngestController.java [hunk 7]
```diff
@@ -57,21 +52,19 @@ public IngestController(IngestService ingestService) {
       ingestService.ingest(fileSize, mimeType, file.getInputStream(), fileName);
     } catch (StoreException | TransformException e) {
       log.warn(
-          "Unable to complete ingest request with params acceptVersion={}, fileSize={}, mimeType={}, title={}, fileName={}",
+          "Unable to complete ingest request with params acceptVersion={}, fileSize={}, mimeType={}, fileName={}",
```
Comment (ahoffer) [inline ingest/src/main/java/com/connexta/ingest/endpoint/IngestController.java]: API change. No more title param.

**Comment by ahoffer** [comment]
build now

File: build.gradle (hunk 5 above)
Comment (ahoffer) [reply build.gradle]: Consolidated repository blocks

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Pull request merged**